File: fix_quotes.js
const fs = require('fs');
let text = fs.readFileSync('js/data/lessons.js', 'utf8');

text = text.replace("title: 'Динамічна пам'ять'", 'title: "Динамічна пам\\'ять"');
text = text.replace("title: 'Класи і об'єкти C++ українською'", 'title: "Класи і об\\'єкти C++ українською"');
text = text.replace("title: 'Класи і об'єкти'", 'title: "Класи і об\\'єкти"');
text = text.replace("title: 'Класи та Об'єкти'", 'title: "Класи та Об\\'єкти"');

fs.writeFileSync('js/data/lessons.js', text, 'utf8');
console.log("Quotes fixed!");

File: fix_quotes.py
import codecs

with open('js/data/lessons.js', 'r', encoding='utf-8') as f:
    text = f.read()

text = text.replace("title: 'Динамічна пам'ять'", 'title: "Динамічна пам\\'ять"')
text = text.replace("title: 'Класи та Об'єкти'", 'title: "Класи та Об\\'єкти"')

with open('js/data/lessons.js', 'w', encoding='utf-8') as f:
    f.write(text)

print("Fixed quotes in lessons.js")
